stripe.go

```diff
@@ -944,7 +945,43 @@ var appInfo *AppInfo
 var backends Backends
 var encodedStripeUserAgent string
 var encodedUserAgent string
-var httpClient = &http.Client{Timeout: defaultHTTPTimeout}
+
+// The default HTTP client used for communication with any of Stripe's
+// backends.
+//
+// Can be overridden with the function `SetHTTPClient` or by setting the
+// `HTTPClient` value when using `BackendConfig`.
+var httpClient = &http.Client{
+	Timeout: defaultHTTPTimeout,
+
+	// There is a bug in Go's HTTP/2 implementation that occasionally causes it
+	// to send an empty body when it receives a `GOAWAY` message from a server:
+	//
+	//     https://github.com/golang/go/issues/32441
+	//
+	// This is particularly problematic for this library because the empty body
```

Comment:
Great comment; thanks for capturing all the context in here.  It's my sincere hope that we can re-enable H/2 support soon, when that Go issue gets addressed.

Reply:
Thanks! And yep, totally +1.

Reply:
Thanks! And yep, totally +1.